### DISCUSSION TITLE
Unable to obtain Px in LengthUnit

### DISCUSSION THREAD

**Discussion by CWH0908** [postcss/postcss#1936] [General]
When my style unit is Px, I see that the type returned by LengthUnit is px. How can I distinguish the unit types of Px?

**Comment by CWH0908** [comment]
Px has been converted to px, and I am unable to distinguish the type of Px during the second packaging process

**Comment by ai** [comment]
PostCSS doesn’t have `LengthUnit`. Do you use some other parser?
